gcp/iam/workload_identity_federation/trusted_issuers.yaml

```diff
@@ -0,0 +1,52 @@
+azure:
```

Comment:
This file is used as a templatefile to substitute the `owner` and `workspace_uuid` values specified